src/app.js

```diff
@@ -1 +1,36 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+const [source, destination] = process.argv.slice(2);
+
+function copyFile(copyFrom, copyTo) {
+  try {
+    fs.existsSync(copyFrom);
+  } catch (error) {
+    console.error(error);
+  }
+
+  if (copyFrom === copyTo) {
```

Comment:
The check `if (copyFrom === copyTo)` is correct in terms of logic, but it might be beneficial to log a message indicating that the source and destination are the same, and therefore no action will be taken.